src/index.js

```diff
@@ -178,7 +178,7 @@ export default function(options = {}) {
                 code: `${codeExportSparse}export default ${injectFnName}(${JSON.stringify(
                   result.css
                 )},${JSON.stringify(codeExportDefault)});`,
-                map: options.sourceMap && result.map
+                map: options.sourcemap && result.map
```

Comment:
maybe `options.sourcemap || options.sourceMap` to not break backward compatibility?